.github/workflows/build_and_test_workflow.yml

```diff
@@ -6,7 +6,7 @@ name: Build and test
 # trigger on every commit push and PR for all branches except pushes for backport branches
 on:
   push:
-    branches: ['**', '!backport/**']
+    branches: ['main', '[0-9].x', '[0-9].[0=9]+'] # Run the functional test on push for only release branches
```

Comment:
It seems like that your commits includes a lot changes which already merged recently, can you rebase your branch, and make sure this PR only have your changes? Thanks

Reply:
I've forced push the branch to remove the duplicated commits, thanks!